events/README.md

```diff
@@ -0,0 +1,75 @@
+# events microservice - Large Events (Knative Proof of Portability)
+
+Add, edit, and fetch events list.
+
+These instructions are specific to this microservice. For general instructions and other information, see [the master README.md](../README.md).
+
+### Installing
+
+How to install and run this microservice for local development.
+
+Each microservice has slightly different dependencies and thus slightly different installation procedures. These instructions are specific to the events microservice, but each other subfolder's README.md has an "Installing" section with more accurate instructions.
+
+Make sure your working directory is this microservice's subfolder.
+
+```sh
+cd events
+```
+
+Set up and activate a virtual environment.
+
+```sh
+python3 -m venv venv && . venv/bin/activate
+```
+
+Install the required python modules.
+
+```sh
+pip3 install -r requirements.txt
+```
+
+Provision a MongoDB instance (e.g. via [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)), then provide the app with its endpoint via an environmental variable.
+
+```sh
+export MONGODB_URI="mongodb+srv://[username]:[password]@[cluster-address]"
+```
+
+### Running the service
+
+To locally run this microservice, execute `app.py` with Python.
+
+```sh
+python3 app.py
+```
+
+## Running the tests
+
+Testing again is handled on the microservice level. To test, ensure your working directory is this microservice's subfolder then use `unittest`.
+
+```sh
+python3 -m unittest discover
+```
+
+To see code coverage, use Coverage.py
+
+```sh
+# install coverage if you don't already have it
+python3 -m pip install --upgrade coverage
+# run coverage on all discovered tests analyzing the 'app' package
+coverage run -m --source app unittest discover
+# report coverage with line numbers
+coverage report -m
+```
+
+## Deployment
```

Comment:
Even deployment is slightly repetitive here - the only difference between how the services are treated is the service name, which is generalized pretty well in the main README.

Reply:
Done https://github.com/knative-portability/large-events/pull/39/commits/927c963ae0baa919fa1d67e7b1c46ed570ee3b29